docs/rest_api.md

```diff
@@ -173,6 +169,92 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, indexed by the name of the network with interior dictionaries indexed by project and channel.
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "driver_id", the id of the network
```

Comment:
it isn't actually the network_id field, it's the label 
the value for 'driver_id' is the network_id field 
this from a previous request by @xuhang57 so he should also comment on changes